configs/trainer.yaml

```diff
@@ -0,0 +1,13 @@
+trainer_kwargs:
+  max_epochs: 1
```

Comment:
want to set these to sensible values for the code that gets merged. Can maintain a local debugging version with lower vals